### PR TITLE
[script.xbmc.unpausejumpback@matrix] 3.0.3

### PR DESCRIPTION
**Pull request by bossanova808**
### Add-on details:

- **General**
  - Add-on name: Unpause Jumpback
  - Add-on ID: script.xbmc.unpausejumpback
  - Version number: 3.0.3
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/bossanova808/script.xbmc.unpausejumpback/
  

        This addon will jump back a number of seconds whenever a video is un-paused - to make sure you don't miss anything.
        You can set the amount of seconds to jump back, and the minimum duration of the pause before the addon will trigger a jump back.
        It also allows to jump back or forward after rewind or fast-forward of a specified amount of seconds to compensate for over-shooting.
        (Originally created by Memphiz, up-keep taken over by bossanova808 in 2020 for Kodi Matrix and beyond).
    

### Description of changes:

v3.0.3
      - Minor bugfix for http/s sources
    

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by bossanova808** [comment]
(Fix of a user submitted fix).  Apparently Python has different precedence (lower) for `or` than `and` - so I read this as evaluating left to right... but it actually needs brackets...

Confirmed working: https://forum.kodi.tv/showthread.php?tid=355778&pid=3069937#pid3069937

File: script.xbmc.unpausejumpback/resources/lib/common.py [hunk 1]
```diff
@@ -190,10 +179,10 @@ def footprints(startup=True):
     :param startup: optional, default True.  If true, log the startup of an addon, otherwise log the exit.
     """
     if startup:
-        log_info(f'Starting...')
-        log_info(f'Kodi Version: {KODI_VERSION}')
-        log_info(f'Addon arguments: {ADDON_ARGUMENTS}')
+        log(f'Starting...', level=xbmc.LOGINFO)
```
Comment (enen92) [inline script.xbmc.unpausejumpback/resources/lib/common.py:182]: Can you log this as debug instead?

**Comment by bossanova808** [comment]
I'd prefer not...with service addons and bug reports, it's super handy to know that the addon has started (and how) - and users are just so woeful with debug logs....can you live with it as is?

**Review by enen92 — APPROVED** [review]
Fine to me

**Pull request merged**